framework/resources/Datasources/schema_change_empty_batch/maprdb/binary/copyFiles.sh

```diff
@@ -5,10 +5,11 @@ set -x
 set -e
 
 if ! $(hadoop fs -test -d ${DRILL_TESTDATA}/schema_change_empty_batch/maprdb/binary/data)
-    then
+then
+        hadoop fs -rm -r -f ${DRILL_TESTDATA}/schema_change_empty_batch/maprdb/binary/data
```

Comment:
Why delete when the folder is not there?

Reply:
Updated. I observed a case where the directory exists, but was empty. The script ran, but the subsequent script failed, since there were no input files. In my opinion, we should always remove and re-create the directory, to avoid such scenarios. Also cases were we update one or more files with the directory - they never take effect other wise.